tests_end2end/functional/.env.example

```diff
@@ -0,0 +1,15 @@
+# .env.example
+# Postgres
+PG_HOST=localhost
+PG_PORT=5432
+PG_DBNAME=tests
+PG_USER=postgres
+PG_PASSWORD=postgres
+
+# Kafka
+KAFKA_CONNECT_URL=http://localhost:8083
+DEFAULT_CONNECTOR_NAME=mosquitto-source-connector
+
+# Tests flags
+USE_EXTERNAL_POSTGIS=false
+E2E_MANUAL_INSPECTION=false
```

Comment:
It is wise to prefix the env vars used by a given component to avoid collisions (pe. PG_HOST could be used by another component in the platform)

I'd suggest to use KAFNUS_

For configurations related with testing maybe the prefix could be speciallized to KAFNUS_TESTS_